lib/puppet/parser/scope.rb

```diff
@@ -883,7 +883,7 @@ def to_s
   # @deprecated use #pop_epehemeral
   # @api private
   def unset_ephemeral_var(level=:all)
-    Puppet.deprecation_warning('Method Parser::Scope#unset_ephemeral_var() is deprecated')
+    Puppet.deprecation_warning(_('Method Parser::Scope#unset_ephemeral_var() is deprecated'))
```

Comment:
`Parser::Scope#unset_ephemeral_var()` contains several function/classname keywords, should we comment about not translating it?